docs/blog/20231004-Daeraxa-OctoberUpdate.md

```diff
@@ -0,0 +1,166 @@
+
+This is not a "Pulsar team" project in the sense that members of the main organization are going to be providing bug fixes or updates in response to logged issues (of course they are free too, just like any other community member). This is an entirely community-led endeavour and we hope everyone will be encouraged to get involved!
+
+This isn't ready _just_ yet, but look out for an update right here on the blog (or any of our other community areas) once we are fully live and ready!
+
+## Modern Tree-sitter blog posts
+
+<img src="./assets/tree-sitter.png" height="150" />[^1]
+
+If you hadn't already seen or read them, [@savetheclocktower] has been writing a series of blog posts that go into detail about all the work that has been going on in Pulsar to support a modern implementation of [Tree-sitter](https://tree-sitter.github.io/tree-sitter/). Parts [one](https://pulsar-edit.dev/blog/20230925-savetheclocktower-modern-tree-sitter-part-1.html) and [two](https://pulsar-edit.dev/blog/20230927-savetheclocktower-modern-tree-sitter-part-2.html) are available right now and keep an eye on the Pulsar blog for more updates!
+
+## Converting PPM's code from callbacks to async
+
+<img src="./assets/development.png" height="150" />
+
+Community member [@Nemokosch/@twocolours]  has been spearheading the conversion of our [PPM codebase](https://github.com/pulsar-edit/ppm) to upgrade old JavaScript callback style to modern ES6 `async/await`.
+
+The PPM repo, while modern and accepted when written, is now mostly viewed as being outdated due to the pervasiveness of deeply nested callbacks. In some instances haveing nesting six levels deep, and as anyone who's been programming in JavaScript prior to ES6 knows, this can be cumbersome, difficult, and frustrating to work with. That is exactly why the effort has been made to convert the entire codebase into a shallow nested, early return patterned, modern ES6 source utilizing async/await properly. In hopes that a greatly simplified codebase can pave the way for easier modifications in the future.
+
+To show what we are talking about, take this (simplified but real) code as an example:
+
+**From this:**
+
+```js
+if (condition) {
+  this.promiseFunc(value, error => {
+    if (error != null) { return callback(error); }
+
+    this.promiseFunc(value, error => {
+      if (error != null) { return callback(error); }
+
+      this.promiseFunc(value, error => {
+        if (error != null) { return callback(error); }
+
+        this.promiseFunc(value, error => {
+          if (error != null) { return callback(error); }
+
+          this.promiseFunc(value, error => {
+            if (error != null) { return callback(error); }
+
+            this.promiseFunc(value, error => {
+              if (error != null) { return callback(error); }
+
+              this.doThing(value);
```

Comment:
```suggestion
              this.doThing(value);
            });
          });
        });
      });
    });
  });
}
```

Reply:
Huh, dunno where the closing brackets went, those are still on my draft one...